tests/NHSD.BuyingCatalogue.Capabilities.API.UnitTests/NHSD.BuyingCatalogue.Capabilities.API.UnitTests.csproj

```diff
@@ -10,15 +10,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="coverlet.msbuild" Version="2.7.0">
+    <PackageReference Include="coverlet.msbuild" Version="2.9.0">
```

Comment:
With this change to coverlet, would you mind making sure the our code coverage tool still works if haven't already?

Reply:
Yep, it still works. However might be worth adding a tech debt ticket to revisit some tests as some classes have 0% on code coverage